build/pmm-client-docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM centos:latest
```

Comment:
it is openshift business requirement to use rhel based image.

Reply:
Maybe it is better to use centos:7 instead of latest?

Reply:
has sense, fixed.